forms/management/commands/generate_areasearch_form.py

```diff
@@ -0,0 +1,678 @@
+        section=contact_section,
+        label="Etunimi",
+        type="textbox",
+        identifier="etunimi",
+        enabled=True,
+        required=False,
+        sort_order=1,
+    )
+    Field.objects.create(
+        section=contact_section,
+        label="Sukunimi",
+        type="textbox",
+        identifier="Sukunimi",
+        enabled=True,
+        required=False,
+        sort_order=2,
+    )
+    Field.objects.create(
+        section=contact_section,
+        label="Sähköposti",
+        type="textbox",
+        identifier="sahkoposti",
+        enabled=True,
+        required=False,
+        sort_order=3,
+    )
+    Field.objects.create(
+        section=contact_section,
+        label="Puhelinnumero",
+        type="textbox",
+        identifier="puhelinnumero",
+        enabled=True,
+        required=False,
+        sort_order=4,
+    )
+
+    invoice_section2 = Section.objects.create(
+        form=form,
+        parent=people_section,
+        title="Laskunsaaja",
+        identifier="laskunsaaja",
+        applicant_type="person",
+    )
+    Field.objects.create(
+        section=invoice_section2,
+        label="Eri kuin hakija",
+        type="checkbox",
+        identifier="eri-kuin-hakija",
+        enabled=True,
+        required=False,
+        sort_order=0,
+        hint_text="""Huomaathan, että Helsingin kaupunki ei lähetä laskuja sähköpostitse.
+                     Täytä vain, jos laskunsaaja on eri kuin hakija.""",
+    )
+    invoice_section2_language = Field.objects.create(
+        section=invoice_section2,
+        label="Kieli",
+        type="dropdown",
+        identifier="kieli",
+        enabled=True,
+        required=False,
+        sort_order=1,
+    )
+
+    Choice.objects.create(
+        field=invoice_section2_language,
+        text="suomi",
+        value="suomi",
+        has_text_input=False,
+    )
+    Choice.objects.create(
+        field=invoice_section2_language,
+        text="ruotsi",
+        value="ruotsi",
+        has_text_input=False,
+    )
+    Choice.objects.create(
+        field=invoice_section2_language,
+        text="englanti",
+        value="englanti",
+        has_text_input=False,
+    )
+    Field.objects.create(
+        section=invoice_section2,
+        label="c/o",
+        type="textbox",
+        identifier="c_o",
+        enabled=True,
+        required=False,
+        sort_order=2,
+    )
+    Field.objects.create(
+        section=invoice_section2,
+        label="Katuosoite",
+        type="textbox",
+        identifier="katuosoite",
+        enabled=True,
+        required=False,
+        sort_order=3,
+    )
+    Field.objects.create(
+        section=invoice_section2,
+        label="Postinumero",
+        type="textbox",
+        identifier="postinumero",
+        enabled=True,
+        required=False,
+        sort_order=4,
+    )
+    Field.objects.create(
+        section=invoice_section2,
+        label="Postitoimipaika",
```

Comment:
Could you fix this typo?

Reply:
I did, sorry for not complying with your previous comment about it